src/util/message.h

```diff
@@ -44,8 +46,51 @@ struct messaget
     template <typename... Args>
     static void println(FILE *f, VerbosityLevel lvl, Args &&...args)
     {
-      if(lvl == VerbosityLevel::Error)
-        fmt::print(f, "ERROR: ");
+      /** 
+		* Newly added code to address issue #886.
+		* It prepends a well formatted timestamp and log level to each LOG message. 
+		* It is transparent to the existing logging function, i.e., log_message.
+		* TODO: It can be implemented using a separate function and invoked here and/or elsewhere when necessary. 		
+		*/
+      struct tm *
+        timestamp_struct; //A time structure consisting of 9 members, such as tm_sec, tm_min, tm_hour, tm_mday, tm_mon, tm_year, etc.
+      constexpr auto timestamp_format = "YYYY-MM-DD HH:MM:SS";
+      constexpr int timestamp_size =
+        std::char_traits<char>::length(timestamp_format) + 1;
+      constexpr auto timestamp_eval = "%Y-%m-%d %H:%M:%S";
```

Comment:
I suggest replacing this with `constexpr std::string timestamp_eval {"%Y-%m-%d %H:%M:%S"};`

Reply:
The same issue as above.